docs/cloud/satellites/byoc/aws.md

```diff
@@ -0,0 +1,116 @@
+# Installing BYOC in AWS
+
+This page documents the requirements and steps required to install BYOC Satellites in AWS.
```

Comment:
```suggestion
This page documents the requirements and steps required to install [BYOC](https://earthly.dev/pricing) Satellites in AWS.
```

Reply:
Should we briefly re-iterate what BYOC is here in case a user lands here and is confused? (and doesn't click the link)

Reply:
Why would we link to pricing?

Reply:
The pricing page has the BYOC plan and explains what it is. It might be useful if someone has landed on this page but hasn't seen the pricing page yet.